test/e2e/gardener/upgrade/upgrade.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package upgrade
+
+import (
+	"context"
+	"os"
+	"time"
+
+	e2e "github.com/gardener/gardener/test/e2e/gardener"
+	"github.com/gardener/gardener/test/framework"
+	shootupdatesuite "github.com/gardener/gardener/test/utils/shoots/update"
+	"github.com/gardener/gardener/test/utils/shoots/update/highavailability"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Gardener upgrade Tests for", Label("gardener"), func() {
+	var (
+		gardenerPreviousRelease = os.Getenv("GARDENER_PREVIOUS_RELEASE")
+		gardenerCurrentRelease  = os.Getenv("GARDENER_NEXT_RELEASE")
+		projectNamespace        = "garden-local"
+	)
+
+	Context("Shoot::e2e-g-upgrade", func() {
+		var (
+			parentCtx  = context.Background()
+			job        *batchv1.Job
+			err        error
+			shootTest2 = e2e.DefaultShoot("e2e-g-upgrade")
```

Comment:
Where is `shootTest{1}`? 😕 

```suggestion
			shootTest = e2e.DefaultShoot("e2e-upgrade")
```